cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/JmsSource.java

```diff
@@ -323,22 +323,22 @@ public JmsProvider getJmsProvider() {
    */
   public static class JmsPluginConfig extends PluginConfig {
     @Name(JMS_DESTINATION_NAME)
-    @Description("Name of the destination to get messages")
+    @Description("Name of the destination from which to get messages.")
     private String destinationName;
 
     @Name(JMS_MESSAGES_TO_RECEIVE)
-    @Description("Max number messages should be retrieved per poll. The default value is 50.")
+    @Description("Max number of messages that should be retrieved per poll. The default value is 50.")
```

Comment:
is there a use for that?  The plugins have different styles due to different authors, but I actually think this is better than defining some variable elsewhere since you can just look at it an know what it does instead of looking in 2 places